packages/subgraph/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "pool-matadata-registry-graph",
+  "license": "UNLICENSED",
+  "scripts": {
+    "codegen": "graph codegen",
+    "build": "graph build",
+    "deploy": "graph deploy --node https://api.studio.thegraph.com/deploy/ pool-matadata-registry-graph",
+    "create-local": "graph create --node http://localhost:8020/ pool-matadata-registry-graph",
+    "remove-local": "graph remove --node http://localhost:8020/ pool-matadata-registry-graph",
+    "deploy-local": "graph deploy --node http://localhost:8020/ --ipfs http://localhost:5001 pool-matadata-registry-graph",
```

Comment:
were you able to deploy locally? can we have a README with `anvil`, `graph-node`, etc, set up?

Reply:
Yep! 🎉

![Captura de Tela 2023-03-03 às 16 09 55](https://user-images.githubusercontent.com/17722915/222806587-7d2a92cd-d9f9-4d8a-ba2a-5ca131d6005e.png)

Reply:
Sure... on linear [BAL-91](https://linear.app/bleu-llc/issue/BAL-91/create-a-readme-with-anvil-graph-node-setup-instructions)